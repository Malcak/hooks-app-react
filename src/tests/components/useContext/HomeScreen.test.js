import React from 'react';
import { mount } from 'enzyme';
import { HomeScreen } from '../../../components/useContext/HomeScreen';
import { UserContext } from '../../../components/useContext/UserContext';

describe('tests on the <HomeScreen /> component', () => {
  const user = { id: '1234', name: 'Fernando', email: 'fernando@mail.com' };
  const wrapper = mount(
    <UserContext.Provider value={{ user: user }}>
      <HomeScreen />
    </UserContext.Provider>
  );
  test('should display the component correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
