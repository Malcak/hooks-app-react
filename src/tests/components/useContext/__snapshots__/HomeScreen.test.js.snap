// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on the <HomeScreen /> component should display the component correctly 1`] = `
<div>
  <h1>
    Home
  </h1>
  <hr />
  <pre>
    {
   "id": "1234",
   "name": "Fernando",
   "email": "fernando@mail.com"
}
  </pre>
</div>
`;
